client/httpclient.go

```diff
@@ -125,6 +125,11 @@ func (c *httpClient) SetAvailableComponents(components *protobufs.AvailableCompo
 	return c.common.SetAvailableComponents(components)
 }
 
+// SetConnectionStatus implements OpAMPClient.SetConnectionStatus
+func (c *httpClient) SetConnectionStatus(status *protobufs.ConnectionSettingsStatus, scheduleSend bool) error {
+	return c.common.SetConnectionStatus(status, scheduleSend)
+}
```

Comment:
Let just use `SetConnectionSettingsStatus` to be consistent with the `ConnectionSettingsStatus` message.